src/fsharp/infos.fs

```diff
@@ -518,6 +518,19 @@ type OptionalArgInfo =
         else 
             NotOptional
 
+type CallerInfoInfo =
+    | NoCallerInfo
+    | CallerLineNumber
+    | CallerMemberName
+    | CallerFilePath
+
+    override x.ToString() =
```

Comment:
In normal F# code `%A` nice-prints union types, but it seems that in the compiler this is not the case?  I forgot about `%+A`, will try that out.